lib/config/proxy.go

```diff
@@ -60,10 +60,12 @@ type ProxyServerOnline struct {
 }
 
 type ProxyServer struct {
-	Addr              string `yaml:"addr,omitempty" toml:"addr,omitempty" json:"addr,omitempty"`
-	PDAddrs           string `yaml:"pd-addrs,omitempty" toml:"pd-addrs,omitempty" json:"pd-addrs,omitempty"`
-	ServerVersion     string `yaml:"server-version,omitempty" toml:"server-version,omitempty" json:"server-version,omitempty"`
-	RequireBackendTLS bool   `yaml:"require-backend-tls,omitempty" toml:"require-backend-tls,omitempty" json:"require-backend-tls,omitempty"`
+	Addr string `yaml:"addr,omitempty" toml:"addr,omitempty" json:"addr,omitempty"`
+	// Addrs will attach server to additional addrs.
+	Addrs             []string `yaml:"addrs,omitempty" toml:"addrs,omitempty" json:"addrs,omitempty"`
```

Comment:
How about reusing `Addr` and separating the addresses by `,`? It's compatible with existing configs.
If we do add `Addrs`, we need to describe it in the official docs. I don't want that...